content/applications/sales/point_of_sale/payment/worldline.rst

```diff
@@ -58,8 +55,9 @@ Hostname
 Port number
 ~~~~~~~~~~~
 
-On the *Port number* screen, enter **"9001"** and click on :menuselection:`OK` (*ECR protocol SSL
-no*) :menuselection:`--> OK`. Click on **Stop** three times; the terminal automatically restarts.
+On the *Port number* screen, enter **9001** (or **9050** for Windows) and click on
```

Comment:
@LoredanaLrpz 
Hey! We were a bit confused by this part of the documentation, what do you mean by (or ... on Windows). You mean, on a virtual IoT box ? Some clients were confused as they thought it was if their computer is on Windows